src/cmd_line/subparser.ts

```diff
@@ -83,7 +82,11 @@ export const commandParsers = {
 
   jumps: {
     abbrev: 'ju',
-    parser: parseJumpsCommandArgs,
+    parser: () => new JumpsCommand(),
+  },
+
+  jump: {
```

Comment:
This shouldn't be necessary - the `abbrev` above should make everything between `ju` and `jumps` parse to `JumpsCommand`

Reply:
Vim handles three aliases: ju, jump and jumps for the same command. In order to follow vim as close as possible I added 'jump'.

Reply:
You'll find that `jum` works as well 🙂 Using `abbrev` makes everything between `abbrev` and the full name work, so without this entry into the map, `jump` will work (as will `jum`)

Reply:
Oh, I didn't know that. That's pretty great. I have removed it now.